packages/@lwc/engine/src/framework/restrictions.ts

```diff
@@ -274,7 +274,6 @@ function getShadowRootRestrictionsDescriptors(
         getElementById: 0,
         getSelection: 0,
         elementsFromPoint: 0,
-        dispatchEvent: 0,
```

Comment:
@t-lau @manuel-jasso We should make sure locker allows dispatchEvent() on SecureTemplate.

Reply:
@arcadeteddy

Reply:
We currently don't allow it. https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.security_locker_global_access I think back then the synthetic ShadowRoot didn't support it either... If it doesn't cause any security concerns, we can whitelist it.

Reply:
yes, let's open a task to track that @arcadeteddy

Reply:
@arcadeteddy can you confirm that this is now allowed?